### DISCUSSION TITLE
[Feature] AI Album/folder sort

### DISCUSSION THREAD

**Discussion by SpiderUnderUrBed** [immich-app/immich#10678] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I made a reddit post about this recently, but I am pretty confident this isn't a existing feature yet, maybe something simular exists, here it is:
In the future, is it planned for immich to be able to sort photos into folders based on like, parameters, for example you could label a folder like "Friends" or "Nature" and immich will sort pictures into those folders/albums that fit the context. Like sorts. Is there any feature like this/planned feature like this?
If not ill file a feature request as this sounds cool and will be SOO useful. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by danieldietzler** [comment]
Workflows will enable that in the mid-term. (https://immich.app/roadmap)

**Comment by bo0tzz** [comment]
#1673

**Reply by SpiderUnderUrBed** [reply]
This feels a big more featureful/diffrent to what that discussion suggested. But my feature would be addressed in some capacity with workflows.
